### DISCUSSION TITLE
[Feature Request][Tab Sleep] Consider adding "Unload All Tabs", "Unload Right/Left Tabs" context menu

### DISCUSSION THREAD

**Discussion by GunGunGun** [Floorp-Projects/Floorp#407] [Feature Request]
### Is your feature request related to a problem?

Tab Sleep is great but sometimes we need to unload a bunch of tabs manually to save RAM in case our computer about to run out of memory for heavy tasks like encoding, gaming...

### Describe the solution you'd like

I would love to have a tab context menu to unload all tabs manually.

### Describe alternatives you've considered

If I remember correctly there's some userChrome.js scripts to do so, I'll try to find them later if it's helpful.

### Additional context

Nothing more.

### Off-topic

Thanks very much for developing Floorp, great web browser. Please keep it up!

**Comment by breadbeforerice** [comment]
+1
